packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -99,3 +99,24 @@
 	background-color: $gray-900;
 	padding: $grid-unit-20 0;
 }
+
+/* In general style overrides are discouraged.
+ * This is a temporary solution to override the InputControl component's styles.
+ * The `Theme` component will potentially be the more appropriate approach
+ * once that component is stabilized.
+ * See: packages/components/src/theme
+ */
+.edit-site-sidebar-navigation-screen__input-control {
```

Comment:
Could we add a temporary override for the help text color too? It's really low contrast against the dark background and barely legible.